jetstream/external_config.py

```diff
@@ -78,7 +79,7 @@ def entity_from_path(path: Path) -> Union[ExternalConfig, ExternalOutcome]:
     if is_outcome:
         platform = path.parent.name
         spec = OutcomeSpec.from_dict(config_dict)
-        return ExternalOutcome(slug=slug, spec=spec, platform=platform)
+        return ExternalOutcome(slug=slug, spec=spec, platform=platform, commit_hash=None)
```

Comment:
We could also set some dummy string here to avoid commit_hash having type `Optional`. But this feels a little cleaner perhaps

Reply:
Agreed, this makes a lot of sense